ogr/ogrsf_frmts/miramon/mm_wrlayr.c

```diff
@@ -6467,6 +6507,34 @@ MMWriteValueToszStringToOperate(struct MiraMonVectLayerInfo *hMiraMonLayer,
     return 0;
 }
 
+int MMWritePreformatedNumberValueToRecordDBXP(
+    struct MiraMonVectLayerInfo *hMiraMonLayer, char *registre,
+    const struct MM_FIELD *camp, const char *valor)
+{
+    if (!hMiraMonLayer)
+        return 1;
+
+    if (!camp)
+        return 0;
+
+    if (MMResizeStringToOperateIfNeeded(hMiraMonLayer,
+                                        camp->BytesPerField + 10))
```

Comment:
why +10 ?

Reply:
To avoid resizing everytime 1 bytes is needed. But it's not important. It could be +1 (for the \0 byte)

Reply:
ok, thanks